Mathlib/Algebra/BigOperators/Pi.lean

```diff
@@ -3,7 +3,7 @@ Copyright (c) 2018 Simon Hudon. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Simon Hudon, Patrick Massot
 -/
-import Mathlib.Algebra.BigOperators.Group.Finset
+import Mathlib.Algebra.BigOperators.GroupWithZero.Finset
```

Comment:
I was gonna ask Yaël if this was okay, but I guess it is 😅

Reply:
`Algebra.BigOperators.Pi` hasn't yet been split up according to `Group`/`GroupWithZero`/`Ring`, so I just don't care yet 😁